### PR TITLE
{data} [intel/2016a] h5py/2.5.0 serial (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
@boegel please review

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
node2618.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6e1fd940373b1cc38b9e for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node2618.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/386acfc21aa89b42b559 for a full test report.

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+name = 'h5py'
+version = '2.5.0'
+
+homepage = 'http://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'usempi': True}
```
Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb]: serial => no MPI?

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+hdf5ver = '1.8.16'
+hdf5versuffix = '-serial'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s%s' % (hdf5ver, hdf5versuffix)
+
+prebuildopts = ' python setup.py configure --hdf5=$EBROOTHDF5 && '
```
Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb]: should this also be done in the other `serial` h5py easyconfigs?

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('HDF5', hdf5ver, hdf5versuffix),
+    ('pkgconfig', '1.1.0', '-Python-%(pyver)s'),
```
Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb]: build dep?

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1-serial.eb [hunk 4]
```diff
@@ -19,10 +19,11 @@ pyver = '%s.10' % py_maj_min
 hdf5ver = '1.8.15-patch1'
 versionsuffix = '-Python-%s-HDF5-%s-serial' % (pyver, hdf5ver)
 
+prebuildopts = ' python setup.py configure --hdf5=$EBROOTHDF5 && '
+
 dependencies = [
     ('Python', pyver),
     ('HDF5', hdf5ver),
-    ('pkgconfig', '1.1.0', '-Python-%(pyver)s'),
```
Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1-serial.eb]: nonono, see https://github.com/hpcugent/easybuild-easyconfigs/pull/2476 :)

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+hdf5ver = '1.8.16'
+hdf5versuffix = '-serial'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s%s' % (hdf5ver, hdf5versuffix)
+
+prebuildopts = ' python setup.py configure --hdf5=$EBROOTHDF5 && '
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('HDF5', hdf5ver, hdf5versuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb]: needs `pkgconfig`, cfr. https://github.com/hpcugent/easybuild-easyconfigs/pull/2476

Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb]: it's a build dep though, see https://github.com/h5py/h5py/blob/master/setup.py#L35

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
node2726.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3f9963093ec9155a5300 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2052.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2c074639e13cec773b9e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2723.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ebc85fce3ac998e7d19a for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in this PR)
node2726.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/30981e09fc331c03c9bb for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 6 out of 6 (3 easyconfigs in this PR)
node2091.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/98f73f6acd129146f3c5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2090.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/49d39e12cceec9fc2e16 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**